extensions/src/main/java/org/sunbird/extension/user/UserExtension.java

```diff
@@ -0,0 +1,17 @@
+package org.sunbird.extension.user;
+
+import java.util.Map;
+
+/**
+ * This interface will provide utility methods to extend user management flows
+ * @author Jaikumar Soundara Rajan
+ *
+ */
+public interface UserExtension {
+	
+	/**
```

Comment:
Rename 'extensionMap' to 'userProfileMap'.

Add comment:
Creates user profile information.
userProfileMap User profile information